oauthenticator/tests/test_globus.py

```diff
@@ -54,14 +54,27 @@ def globus_client(client):
     return client
 
 
+@fixture
+def user():
+    class User:
+        @gen.coroutine
+        def get_auth_state(self):
+            return {'tokens': {}}
+
+        save_auth_state = Mock()
+
+        name = 'wash'
+    return User()
+
+
 @mark.gen_test
 def test_globus(globus_client, mock_globus_sdk):
     authenticator = GlobusOAuthenticator()
     handler = globus_client.handler_for_user(user_model('wash'))
-    name = yield authenticator.authenticate(handler)
-    assert name == 'wash'
-    assert list(authenticator.globus_data['tokens'].keys()) == \
-        ['transfer.api.globus.org']
+    data = yield authenticator.authenticate(handler)
+    assert data['name'] == 'wash'
+    tokens = list(data['auth_state']['tokens'].keys())
+    assert tokens == ['transfer.api.globus.org']
```

Comment:
Are these changes needed because of the change in the return type of `authenticate`?

Reply:
Yes.